scripts/deckbuilder.js

```diff
@@ -720,7 +720,7 @@ var sortDeck = function ()
         if (compare) return compare;
         compare = (cardA.type - cardB.type);
         if (compare) return compare;
-        compare = compareByID(unitA, unitB);
```

Comment:
This was an intentional decision to deviate from the in-game behavior to group copies of the "same" card together.